tree/dataframe/inc/ROOT/RCsvDS.hxx

```diff
@@ -75,7 +77,8 @@ protected:
    std::string AsString();
 
 public:
-   RCsvDS(std::string_view fileName, bool readHeaders = true, char delimiter = ',', Long64_t linesChunkSize = -1LL);
+   RCsvDS(std::string_view fileName, bool readHeaders = true, char delimiter = ',', Long64_t linesChunkSize = -1LL,
+          std::string fillValue = "NaN", unsigned int lookDownForType = 10U);
```

Comment:
Also, what we are trying to address here is non-specified values, so these are rather NA (not available) instead of NaN, should we perhaps have a default that is called `"NA"`?